tip_tracker/lib/core/auth/login/LobbyPage.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:tip_tracker/core/auth/login/Log_in_form.dart';
+import 'package:tip_tracker/core/auth/login/Sign_up_form.dart';
+
+class LobbyPage extends StatefulWidget {
+  const LobbyPage({Key? key}) : super(key: key);
+
+  @override
+  State<LobbyPage> createState() => _LobbyPageState();
+}
+
+class _LobbyPageState extends State<LobbyPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xff183A37),
+      body: SafeArea(
+        child: Center(
```

Comment:
not sure if this is a flutter thing, but do these have to be named child, or can they be named something more descriptive?

Reply:
it does have to be child. pretty much every widget is just a parent of another widget.